### DISCUSSION TITLE
Default settings for repositories

### DISCUSSION THREAD

**Discussion by 3N4N** [community/community#33481] [Other Feature Feedback, Questions, & Ideas] [answered]
Under Features section of a repository settings, there are several, well, features.

o Wikis
o Issues
o Sponsorship
o Projects
o Preserve this repository
o Discussions

The user has to change these features for each of their repositories. But sometimes they may need to enable or disable one or more of these features for _all_ their repos. A simple button below the Features section titled "Set these options for all repositories" would be really helpful.

Also, the default values of these features should be set-able. The same features page can be added in the user settings where they can change these features settings.

---

Totally unrelated but, um, can a GitHub employee change these features for all my repos? If yes, how do I ask?

**Comment by simonnchong** [comment, marked as answer]
I dont think they provide such services
